allure-attachments/src/main/java/io/qameta/allure/attachment/http/HttpRequestAttachment.java

```diff
@@ -53,6 +55,17 @@ public HttpRequestAttachment(final String name, final String url, final String m
         this.cookies = cookies;
     }
 
+    public HttpRequestAttachment(final Builder builder, final Map<String, String> formParams) {
```

Comment:
The old constructor should be calling the new one (providing `null` as a last argument)